added nervos blog post (#178)
leapdao/leapdao-website: src/posts/nervos-announcement.md

---
template: post.html

> title: "Got Scaling? Building layer-2 for Nervos CKB"

Author: The idea was for it to be a snowclone on Got Milk?

> title: "Got Scaling? Building layer-2 for Nervos CKB"

Author: I read `goto considered harmful` a couple of days ago and thought it would be fun to try to use a phrase myself.

> title: "Got Scaling? Building layer-2 for Nervos CKB"

Reviewer: a snowclone on this got milk? https://www.nytimes.com/2014/11/18/upshot/got-milk-might-not-be-doing-you-much-good.html

wouldn't the title need to be rather: `gas-storms considered harmful: building a layer-2 on top of Nervos CKB.`

> title: "Got Scaling? Building layer-2 for Nervos CKB"

Author: that is actually a pretty cool title, doing a snowclone on that is amazing <3

> date: 2020-07-09 17:00:00

Author: ```suggestion
date: 2020-07-30 17:00:00
```

image: ckb.png
author:
  name: "@KenoBudde"
  twitter: "@KenoBudde"
  link: https://github.com/clearwood/
description: Announcement, building layer-2 for Nervos CKB
---

We are excited to announce that we successfully applied for a [grant of the Nervos foundation](https://talk.nervos.org/t/ckb-sidechain-framework/4722) to work on second layer scaling for this blockchain.

## A new kind of challenge

> The focus of Ethereum has been application development in the past, not necessarily scalability. The first turing-complete chain was a breakthrough but the high costs associated with operating on the main-net at the moment means that some ideas are not viable to be operated on it.

Reviewer: the paragraph seems unfinished. it is missing a conclusion, especially in relationship to the headline. what is the new challenge, if ethereum only caters to non-scalable dapps?

> We are actively working on roll-up based solutions for Ethereum and will apply the same tactic towards Nervos. We find the transaction model interesting and want to apply our findings to a new ecosystem. We have been in contact with the part of the Nervos' team for years and are excited to work together with them.

Reviewer: > our findings

our findings of what?

> We are and will be working on roll-up based solutions for Ethereum and will apply the same tactic towards Nervos. We find the transaction model interesting and want to apply our findings to a new ecosystem. We have been in contact with the part of the Nervos' team for years and are excited to work together with them.

Reviewer 2: `We are actively working` removes double `and` in sentence

> We are and will be working on roll-up based solutions for Ethereum and will apply the same tactic towards Nervos. We find the transaction model interesting and want to apply our findings to a new ecosystem. We have been in contact with the part of the Nervos' team for years and are excited to work together with them.

Author: ```suggestion
We are actively working on roll-up based solutions for Ethereum and will apply the same tactic towards Nervos. We find the transaction model interesting and want to apply our findings to a new ecosystem. We have been in contact with the part of the Nervos' team for years and are excited to work together with them.
```

Nervos has been created with its primary focus on scalability, it is a layer 1 that is intended to be used by layer 2 networks, similarly to the beacon chain of ETH 2.0.

We became interested in contributing our knowledge of layer 2 scalability to a network that has been setup to accomodate layer 2 solutions from the get-go.

Interesting for us are especially the [UTXO model employed by Nervos](https://xuejie.space/2020_03_20_what_do_we_mean_when_we_say_account_model/)

> - UTXO model puts less logic in the chain.

Reviewer: ```suggestion
- it puts less logic in the chain.
```

> - UTXO model allows parallel execution.

Reviewer: ```suggestion
- it allows parallel execution.
```

- Generators can fill in the needed features to enable smart contracts.

> The Nervos Cell model is especially interesting, as it implements a state rent through inflation of CKBytes.

Reviewer: > Nervos Cell model

should be explained before used in the text

> We want to provide a simple entry point for applications to run on Nervos, therefore we will spend our time on developing a second layer for Nervos. As majority of todays smart contract developers are knowledgeable with Ethereum and its dev tools we have set out to create a sidechain where these skills can be applied and eventually existing decentralized applications can easily be migrated.

Reviewer: > developing a second layer for Nervos

colloquial, undifferentiated expression. Why not actually use wording from our grant application: https://talk.nervos.org/t/ckb-sidechain-framework/4722

Reviewer: structure is this paragraph is confusing. maybe talk about smart contract developers first, then conclude a simple entry point for those developers to the nervos ecosystem from that.

> We want to provide a simple entry point for applications to run on Nervos, therefore we will spend our time on developing a second layer for Nervos. As majority of todays smart contract developers are knowledgeable with Ethereum and its dev tools we have set out to create a sidechain where these skills can be applied and eventually existing decentralized applications can easily be migrated.

Reviewer: ```suggestion
We want to provide a simple entry point for applications to run on Nervos, therefore we will spend our time on developing a second layer for Nervos. As majority of todays smart contract developers are knowledgeable with Ethereum and its dev tools. We have set out to create a sidechain where these skills can be applied and eventually existing decentralized applications can easily be migrated.
```

> ## Nervos design philosophy that we jive with

Reviewer: everything in this paragraph has already been mentioned in the previous one.

We really nerd out about the following Nervos features.

1. We love that the UTXO-model borrows itself to functional programming and is scalable through [parallel processing](https://medium.com/@sunflora98/utxo-vs-account-balance-model-5e6470f4e0cf).
2. Nervos was built with layer-2 in mind, it lends flexibility to off-chain constructions.

## What will the world get from this Grant?

We believe that it should be easy to work with great technology. Therefore using this grant we will not only work on an EVM compatible side-chain but also provide tools Ethereum developers are already comfortable with. These include the following:

> 1. A WebUI to define UDT or ERC20 token

Reviewer: > tools Ethereum developers are already comfortable with

UDT is specific to CKB, hence not known to Eth developers.

> 2. A CLI tool for deployment to CKB or the sidechain

Reviewer: also CLI tools are specific to our product. The grant proposal specifically mentions that the CLI will be replaced by a UI in later stages, so not clear why it is pointed out here.

3. SDK tools to interact with the sidechain.

> In doing so, releasing to our sidechain will seem very familiar to everyone already working with dApps. Starting from scratch our tools will give you an easy start and decrease time to market for dApps.

Reviewer: > releasing to our sidechain will

what does this mean?

Reviewer: > Starting from scratch our tools

gramar? meaning?

---
template: post.html
title: "Gas-Storms Considered Harmful: Building a Layer-2 on top of Nervos CKB"

> date: 2020-07-09 17:00:00

Author: ```suggestion
date: 2020-08-06 17:00:00
```

---
template: post.html
title: "Got Scaling? Building layer-2 for Nervos CKB"
date: 2020-07-09 17:00:00
image: ckb.png
author:
  name: "@KenoBudde"
  twitter: "@KenoBudde"
  link: https://github.com/clearwood/
description: Announcement, building layer-2 for Nervos CKB
---

> We are proud to announce to have entered a [grantee relationship with the Nervos Foundation](https://talk.nervos.org/t/ckb-sidechain-framework/4722) to work on second layer scaling, a method to increase throughput by bundling transactions on a different network, for CKB. Transferring the advancements made in the Ethereum ecosystem will bring Nervos CKB to the next level of their scalability roadmap.

Reviewer: ```suggestion
We are proud to announce that we have entered a [grantee relationship with the Nervos Foundation](https://talk.nervos.org/t/ckb-sidechain-framework/4722). 

We will work on layer-2 scaling solution for the Nervos ecosystem. Utilizing the concept of rollups,  a method to increase throughput by bundling transactions we hope to bring Nervos CKB to the next level of its scalability roadmap.
```

---
template: post.html

> title: "Got Scaling? Building layer-2 for Nervos CKB"

Author: ```suggestion
title: "gas-storms considered harmful: building a layer-2 on top of Nervos CKB"
```

> title: "gas-storms considered harmful: building a layer-2 on top of Nervos CKB"

Reviewer 3: Title leaves me confused...

We can keep it simple:
Announcement: LeapDAO building a layer2 for Nervos
